spec/jobs/update_account_invoices_spec.rb

```diff
@@ -417,8 +417,6 @@ def travel_to(time)
           [:address1, :address2, :city, :zipcode, :state_id, :country_id].each do |attr|
             expect(invoice_order.billing_address[attr]).to eq july_billable_period2.enterprise.address[attr]
           end
-          expect(invoice_order.billing_address.firstname).to eq "Anna"
```

Comment:
Why has this been removed?

Reply:
This was part of another spec that was changed previously due to the contact name field being removed.

Reply:
Okay, it would be really good to put changes that belong together into the same commit or explain the relationship between the commits in the message.

Reply:
I've rebased and moved the change into the original commit.